Tests/RosalindTests/__Snapshots__/RosalindAcceptanceTests/apple_ios_app.1.json

```diff
@@ -0,0 +1,59 @@
+{
+  "app" : {
+    "children" : [
+      {
+        "unknown" : {
```

Comment:
Changing this to a struct from an enum will imho also make the JSON structure easier to parse.

Reply:
I agreed. Let's see what are the options to continue using the library while having structs with indirect values.